### PR TITLE
Add Issue forms 

### PR DESCRIPTION
**Pull request by mymphe**
This PR add issue forms to the repo

### CODE REVIEW

**Review by TheDZhon — APPROVED** [review]
:heart:

File: .github/ISSUE_TEMPLATE/CONTRACT-BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Contract Bug Report
+description: File a bug report in a contract.
```
Comment (TheDZhon) [inline .github/ISSUE_TEMPLATE/CONTRACT-BUG-REPORT.yml:2]: Should refer to the SECURITY page first :warning: 

Reason: to file a bug by 3rd-party via Immunefi

**Review by TheDZhon — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/CONTRACT-BUG-REPORT.yml (hunk 1 above)
Comment (tamtamchik) [reply .github/ISSUE_TEMPLATE/CONTRACT-BUG-REPORT.yml:2]: It looks like it is impossible to add a link to the GitHub issue reporting interface. Documentation refers

Screenshot 2024-08-26 at 17 11 31

**Comment by tamtamchik** [comment]
Using just a single text box is way easier than filling out an entire form. It lets users think less, saves time, and they can just write what they need. I really think simplicity is the way to go for reporting. But throwing in some markdown guidance could be helpful to keep the report on point and not just a random ramble.

**Review by tamtamchik — APPROVED** [review]

**Review by TheDZhon — APPROVED** [review]
Let the sunshine in

**Pull request merged**